src/vaadin-text-field-mixin.html

```diff
@@ -471,20 +481,31 @@
       node.removeEventListener('blur', this._boundOnBlur);
     }
 
+    _addAutoselectListener(node) {
```

Comment:
It would be better to reuse existing `_addInputListeners` and `_removeInputListeners` methods above, and add corresponding lines there.

Reply:
Done.